lib/smart_answer_flows/calculate-state-pension/calculate_state_pension.govspeak.erb

```diff
@@ -7,15 +7,18 @@
 <% end %>
 
 <% content_for :body do %>
-  Calculate when you’ll reach State Pension age or Pension Credit qualifying age and how much you may get in today’s money for your basic State Pension.
+  Calculate when you’ll reach State Pension age, Pension Credit qualifying age or fine out when you'll be eligible for free bus travel.
 
-  %Get a [State Pension Statement](/state-pension-statement) instead of using this calculator if you're 55 or over and making a financial decision based on [the new pension options](https://www.pensionwise.gov.uk/).%
+  Get a [State Pension Statement](/state-pension-statement) instead of using this calculator if:
+  
+  - you're 55 or over
+  - you're making a financial decision based on [the new pension options](https://www.pensionwise.gov.uk/)
 <% end %>
 
 <% content_for :post_body do %>
   ##What you need to know:
 
-  This calculator gives an estimate of your basic State pension and information about the [new State Pension](/new-state-pension) if you reach State Pension age on or after 6 April 2016.
+  This calculator gives an estimate of your basic State pension and information about the [new State Pension](/new-state-pension).
```

Comment:
Yeah, technically it should as well. Though it's been wrong for awhile!

On 5 October 2015 at 14:27, Isabell Long <notifications@github.com> wrote:

> In
> lib/smart_answer_flows/calculate-state-pension/calculate_state_pension.govspeak.erb
> <https://github.com/alphagov/smart-answers/pull/1989#discussion_r41142748>
> :
>
> >  <% end %>
> >
> >  <% content_for :post_body do %>
> >    ##What you need to know:
> >
> > -  This calculator gives an estimate of your basic State pension and information about the [new State Pension](/new-state-pension) if you reach State Pension age on or after 6 April 2016.
> > +  This calculator gives an estimate of your basic State pension and information about the [new State Pension](/new-state-pension).
>
> Should "pension" be capitalized for consistency with all other occurrences
> of "State Pension" in this sentence?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/smart-answers/pull/1989/files#r41142748>.
>



-- 
[image: Cabinet Office - Government Digital Service]

*Liz Lutgendorff*
Content Analyst
Government Digital Service
Aviation House
London WC2B 6NH